examples/plot_grid_search_census.py

```diff
@@ -45,9 +45,10 @@
 # We can now load and inspect the data by using the `fairlearn.datasets` module:
```

Comment:
this doesn't have any thresholdoptimizer, can we hold off for another PR?

Reply:
somehow the `flake8` check fails on this PR for these, and not on `main`. Not sure what's happening. Trying to get the CI green here.

Reply:
i know... very odd.

Reply:
found the culprit in `setup.cfg`.